orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -115,6 +116,9 @@ public void init() throws DBException {
 
   @Override
   public void cleanup() throws DBException {
+    // Set this thread's db reference (needed for thread safety in testing)
+    ODatabaseRecordThreadLocal.INSTANCE.set(db);
```

Comment:
we have #571 to cover correcting concurrency in the orientdb binding, so sounds fine.